src/lib/util.ts

```diff
@@ -292,20 +292,60 @@ export function transformDiscordLocale(discordLocale: string): string {
 	else return "en"
 }
 
-export function updateButtonColors(row: MessageActionRow, page: number, pages: unknown[]) {
-	if (page === 0) {
-		row.components.forEach(button => {
-			if (button.customId === "first" || button.customId === "previous") (button as MessageButton).setStyle("SECONDARY").setDisabled(true)
-			else (button as MessageButton).setStyle("SUCCESS").setDisabled(false)
-		})
-	} else if (page === pages.length - 1) {
-		row.components.forEach(button => {
-			if (button.customId === "last" || button.customId === "next") (button as MessageButton).setStyle("SECONDARY").setDisabled(true)
-			else (button as MessageButton).setStyle("SUCCESS").setDisabled(false)
-		})
-	} else row.components.forEach(button => (button as MessageButton).setStyle("SUCCESS").setDisabled(false))
+export function getButtonControlLocalizations(func: GetStringFunction) {
+	return {
+		firstLabel: func("pagination.first", { file: "global" }),
+		previousLabel: func("pagination.previous", { file: "global" }),
+		nextLabel: func("pagination.next", { file: "global" }),
+		lastLabel: func("pagination.last", { file: "global" }),
+	}
+}
```

Comment:
do we really need this to be a function? IMO it would look better on either the function that creates the buttons or on the command files themselves. Also please sort these new methods alphabetically regardless